bin/run_tests

```diff
@@ -217,6 +226,7 @@ test_rust_sdk() {
 }
 
 test_integration() {
+    run_docker_test test_batch_injection_events
```

Comment:
Adding the test here causes it to be ran twice. It should either be added here with the coverage line, or put in the test_batch_injection_events() section, not both.

Reply:
Your are completely right, I'll remove it from integration section, thank you